modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
+
+[id="nw-ovn-kubernetes-limitations_{context}"]
+= OVN-Kubernetes limitations
+
+The OVN-Kubernetes default Container Network Interface (CNI) network provider has a limitation that is related to traffic policies.
+The network provider does not support setting the external traffic policy  or internal traffic policy for a Kubernetes service to `local`.
+The default value, `cluster`, is supported for both parameters.
+This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
```

Comment:
~Isn't external IP literally `externalIP`?~ It's `externalIPs` which is kind of awkward.

Reply:
Because external IP is in a distinct field from the type, I thought to cover it somewhat distinctly.